README.md

```diff
@@ -107,7 +95,51 @@ When using the mock-compiler service you will get the same PDF and compilation l
 The idea behind the mock compiler service is to let you work through the entire submission process workflow
 without the hassle of installing the compiler service.
 
-## Quick start using arXiv Compiler service (with actual TeX compilation)
+## Quick start, adding the arXiv Compiler service with full TeX compilation
+
+This builds on the "Quick start with mock compilation" steps above.
+
+How to chose the option that's right for you: If you are not concerned with TeX
+compilation, or you have limited disk space, you will want the mock compiler option.
+If you are concerned with evaluating TeX compilation output and/or the submission
+'process' step under normal operating conditions you will want to install the
+actual Compiler service's converter image.
+
+###  AWS settings and Credentials
+
+One of the back-end services we run is 'localstack' which is a fully functional
+local AWS stack. While this local AWS stack implementation doesn't actually
+check credentials you must have the environment variables set for the service
+to work properly.
+
+First, you will need credentials to AWS ECR to get the converter docker
+image. You should only need to do this once. If it seems your quick
+start is not running the compiler it is probably because it cannot
+access this docker image on AWS ECR.
+
+To do this,
```

Comment:
These are a nice addition.

Reply:
Thanks, though I'm mostly reordering the information. Looks like most of this content was written by David and you!